R/analysis_skeleton.R

```diff
@@ -45,14 +45,16 @@ analysis_skeleton <- function(path = ".", git_init = TRUE, git_clone = NULL,
                               coc_email = getOption("bcgovr.coc_email"),
                               copyright_holder = "Province of British Columbia") {
 
-#   now <- Sys.time()
-#   options(error = quote(error_cleanup(now)))
-#   on.exit(options(error = NULL), add = TRUE)
-  
-  if (path != ".") dir.create(path, recursive = TRUE)
+
+  ## Create directory is path is not current working directory
+  ## Suppress warning if directory is already there. 
+  if (path != ".") tryCatch(dir.create(path, recursive = TRUE),
```

Comment:
Can do `dir.create(path, recursive = TRUE, showWarnings = FALSE)` here instead of tryCatch